homeassistant/config_entries.py

```diff
@@ -3187,6 +3187,76 @@ def async_create_entry(  # type: ignore[override]
 
         return result
 
+    @callback
+    def _async_update(
```

Comment:
I think we should use double underscore if we don't want this to be called by classes implementing `ConfigFlow`